content/core-features/history.textile

```diff
@@ -0,0 +1,19 @@
+---
+title: History
+section: core-features
+index: 21
+---
+
+Ably, by default, retains messages for a duration of 2 min. Within this period, messages are re-sent to the clients expecting to receive them, but could not do so, perhaps due to a short disconnection. Even in such a case, Ably ensures all the backlog messages sent by publishers within these two minutes are sent to the respective recipient clients in the same order as they were received. This holds true for messages relevant to both "@Channel@":/realtime/channels-messages as well as "@Presence@":/realtime/presence objects.
```

Comment:
"min" -> "mins" or "minutes".

I'd be tempted to rephrase this intro to make it clearer how it links to History (clearly stating this is history doing this stuff). A quick example of what I mean would be:

By default, all messages sent on Ably will be stored for 2 minutes on our servers. This allows for clients who disconnect for less than 2 minutes to recover any messages they've missed through our History API. The recovering client will receive these messages in the original order they were sent, and this is applicable to both "@messages@":/realtime/channels-messages#messages and @Presence messages@:"/realtime/presence".